contrib.json

```diff
@@ -372,5 +379,53 @@
         [ "git branch -D {{pr.user.login}}-{{pr.head.ref}}",                   "Delete the local branch used for merging" ]
       ]
     }
+  },
+
+  "stage": {
+    "patch": {
+      "release_type": "patch",
+      "description": "Get the repo ready for releasing a patch",
+      "steps": [
+        { "include": "update stable" },
+        { "include": "update patch" },
+        [ "git merge stable", "Merge stable into patch to stage patch release" ],
+        [ "git push upstream path", "Push upstream to trigger patch release" ],
+        [ "git push origin path", "Update origin" ]
+      ]
+    }
+  },
+  "codeship": {
+    "run": {
+      "steps": [
```

Comment:
These steps basically assume that origin is the videojs repo instead of upstream.